file_paths.md

```diff
@@ -204,6 +217,20 @@ user or group.
             run                               # :rundir                      same
                 agent.pid                     # :pidfile
 
+    C:\ProgramData\PuppetLabs\pxp-agent *
+        etc *
+            pxp-agent.cfg                     # pxp-agent configuration file
+            modules.d *                       # directory where pxp-agent modules are stored
+                puppet.cfg *                  # ../../var/modules/puppet configuration
+        var *
+            log
+                pxp-agent.log                 # enabled by default
+            modules *
```

Comment:
Configuration data should definitely live under `C:\ProgramData\PuppetLabs`.

If scripts are a write-once sort of thing, it could make sense to put them in a global `\Program Files\` rooted directory.  But on the flip side it could be easier to manage a single path if they're all lumped in together (like MCO).  I also don't have a really strong opinion on that aspect, but the more secure option is to place executables under `\Program Files\`.

My only strong opinion here is that Windows mirror the same intent as *nix.